website/docs/docs/collaborate/govern/project-dependencies.md

```diff
@@ -104,6 +104,12 @@ with monthly_revenue as (
 
 For more guidance on how to use dbt Mesh, refer to the dedicated [dbt Mesh guide](/best-practices/how-we-mesh/mesh-1-intro).
 
+### Protecting production
+
+When working in a Development environment, cross-project `ref`s normally resolve to the Production environment of the project. To protect production data, set up a [Staging deployment environment](/docs/deploy/deploy-environments#staging-environment) in your projects. When a staging environment is present in the project, when other projects reference it during development workflows, they resolve to the Staging environment. This adds a layer of security between your Deployment and Production environments by limiting access to production data.
+
+Read [Why use a staging environment](/docs/deploy/deploy-environments#why-use-a-staging-environment) for more information about the benefits. 
```

Comment:
**Highest priority**

An important thing for us to note/callout here: If you maintain a project, you're interested in setting up a Staging environment, and other downstream projects are already depending on it, we recommend that you act in the following order:
1. Create a new environment, but do NOT mark it as "Staging"
2. Define a job in that environment
3. Trigger it to run, and ensure it completes successfully
4. Finally, update the environment to mark it as "Staging"

Otherwise, the time between creating the environment and completing the first successful job run will cause downtime for downstream developers. As soon as the Staging environment exists in a project, dbt Cloud will start using it to resolve cross-project references from downstream projects, and there's no "fail-over" to Production.

We could reflect that here and/or in the "Prerequisites" section above, which currently reads:
```
- Have a deployment environment in the upstream ("producer") project [that is set to be your production environment](/docs/deploy/deploy-environments#set-as-production-environment)
- Have a successful run of the upstream ("producer") project
```

Reply:
I added this in both places. Feel free to change anything tomorrow!